resources/js/components/mod.tsx

```diff
@@ -40,18 +41,84 @@ function settingsLabel(modJson: NonNullable<typeof modNames[string]>, scoreModJs
     : ` (${settings.join(', ')})`;
 }
 
+function getExtendedContent(scoreModJson: ScoreModJson): string | null {
+  const settings = Object.entries(scoreModJson.settings ?? {});
+  switch (scoreModJson.acronym) {
+    case 'HT':
+    case 'DC':
+    case 'DT':
+    case 'NC':
+    {
+      const speedChange = settings.find((s) => s[0] === 'speed_change');
+      return speedChange != null ? `${formatNumber(speedChange[1] as number, 2)}x` : null;
+    }
+
+    case 'DA':
+    {
+      const displayCandidates = {
+        approach_rate: {
+          acronym: 'AR',
+          significantDigits: 1,
+        },
+        circle_size: {
+          acronym: 'CS',
+          significantDigits: 1,
+        },
+        drain_rate: {
+          acronym: 'HP',
+          significantDigits: 1,
+        },
+        overall_difficulty: {
+          acronym: 'OD',
+          significantDigits: 1,
+        },
+        scroll_speed: {
+          acronym: 'SS',
+          significantDigits: 2,
+        },
+      };
+
+      let displayCandidate;
+      let displayValue: number | undefined;
+      for (const [setting, value] of settings) {
+        if (setting in displayCandidates) {
+          if (displayCandidate != null) {
+            return null;
+          }
+
+          displayCandidate = displayCandidates[setting as keyof typeof displayCandidates];
+          displayValue = value as number;
+        }
+      }
+
+      if (displayCandidate != null && displayValue != null) {
+        return `${displayCandidate.acronym}${formatNumber(displayValue, displayCandidate.significantDigits)}`;
```

Comment:
I wonder if this (and the other one) should always use english number formatting...